packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { settingsPageWatchAccountUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { watchMultisigs } from '../fixtures/watchAccounts/watchMultisigs'
+import convertRococoAddressToKusama from '../utils/convertRococoAddressToKusama'
+
+const {
+  publicKey: multisigPublicKey,
+  address: multisigAddress,
+  pureAddress: multisigPureAddress,
+  name: multisigName
+} = watchMultisigs['multisig-with-pure']
+
+describe('Network can be switched', () => {
+  it('should switch account using menu', () => {
+    cy.visitWithLocalStorage({
+      url: settingsPageWatchAccountUrl,
+      accountNames: { [multisigPublicKey]: multisigName as string },
+      watchedAccounts: [multisigPublicKey]
+    })
+
+    cy.url().should('contain', `network=rococo&address=${multisigPureAddress}`)
```

Comment:
We don't want to impose the order of things here. Let's do 2 separate checks
```suggestion
    cy.url().should('contain', 'network=rococo')
    cy.url().should('contain', `address=${multisigPureAddress}`)
```

Reply:
For me both ways are fine